bin/helpers/migrator.mjs

```diff
@@ -0,0 +1,46 @@
+import settings from "../utils/settings.mjs";
+import {files} from "./files.mjs";
+import path from "path";
+
+/**
+ *
+ * SUPER IMPORTANT WHEN CREATING A NEW VERSION
+ * Change the settings.mjs file in the utils folder to reflect the new version you created
+ *
+ */
+const migrate = from => {
+  return new Promise((resolve, reject) => {
+    const currentVersion = settings.version;
+    let configVersion = from;
+    let config = {...files.readSettingsJson()};
```

Comment:
Little pedantic but I don't think you need to wrap .readSettingsJson in an object and then spread it. I think returning the object is good enough here.